### DISCUSSION TITLE
Dynamically generate hashPassword for each dex deployment

### DISCUSSION THREAD

**Discussion by vinayK34** [dexidp/dex#3013] [Q&A]
Hi,
I am looking to generate hashPassword dynamically using bash script which executes a python script inside it and store a value in some variable. Wanted to know how can I pass this variable value to config-map.yaml? 
<img width="207" alt="image" src="https://github.com/dexidp/dex/assets/99893515/05f37a82-f0ae-43ae-93fb-8b9240a95908">

**Comment by nabokihms** [comment]
If you are running Dex with the official container image, you can use env variables to pass hashes like in the following example
https://github.com/dexidp/dex/blob/master/config.docker.yaml
